### PR TITLE
2/2 Feat: menu bar

### PR DESCRIPTION
**Pull request by MathiasDuus**

### CODE REVIEW

**Comment by MathiasDuus** [comment]
[GUI - Menu bar](https://trello.com/c/PWUKXTVB/22-gui-menu-bar)

File: lib/misc/api/api_logon.dart [hunk 1]
```diff
@@ -128,8 +89,8 @@ Future<bool> login(
       debugPrint(response.body);
     }
 
-    // Uses API library to store token
-    await storeToken(
+    // Saves teh new tokens
```
Comment (Setanna) [inline lib/misc/api/api_logon.dart:92]: Minor typo

File: lib/misc/api/api_url.dart [hunk 2]
```diff
@@ -15,7 +15,7 @@ getApiBaseUrl() {
 getBaseUrl() {
   // If debug mode is active, use the dev path.
   if (kDebugMode) {
-    return "http://$devIpAddress";
+    return "http://$ipAddress";
```
Comment (Setanna) [inline lib/misc/api/api_url.dart:18]: Should stay devIpAddress

File: lib/misc/api/api_url.dart [hunk 3]
```diff
@@ -6,7 +6,7 @@ const String devIpAddress = '127.0.0.1:8000';
 getApiBaseUrl() {
   // If debug mode is active, use the dev path.
   if (kDebugMode) {
-    return "http://$devIpAddress/api";
+    return "http://$ipAddress/api";
```
Comment (Setanna) [inline lib/misc/api/api_url.dart:9]: Should stay devIpAddress

**Review by Setanna — CHANGES_REQUESTED** [review]

File: lib/misc/api/api_url.dart (hunk 2 above)
Comment (MathiasDuus) [reply lib/misc/api/api_url.dart:18]: Yes, thank you.

File: lib/misc/api/api_token.dart [hunk 4]
```diff
@@ -42,24 +45,25 @@ Future<String?> getToken() async {
...
   return Future.value(value);
 }
 
+/// Get wheter to auto login from storage
 Future<bool> getAutoLogin() async {
   bool value = await storage.read(key: _autoLogin) == 'true';
   return value;
 }
 
+/// Get wheter user is admin from storage
```
Comment (Setanna) [inline lib/misc/api/api_token.dart:60]: Minor typo, check all for similar typo wheter -> whether

**Review by Setanna — CHANGES_REQUESTED** [review]

File: lib/misc/api/api_token.dart (hunk 4 above)
Comment (MathiasDuus) [reply lib/misc/api/api_token.dart:60]: oopsies

**Review by Setanna — APPROVED** [review]

**Pull request merged**